int-ui/int-ui-standalone/src/main/kotlin/org/jetbrains/jewel/intui/standalone/StandalonePainterHintsProvider.kt

```diff
@@ -83,19 +84,25 @@ public class StandalonePainterHintsProvider(
 
         if (adjustedKey !in supportedCheckboxKeys) {
             if (inDebugMode) {
-                println("${if (isDark) "Dark" else "Light"} theme: color key $key is not supported, will be ignored")
+                JewelLogger
+                    .getInstance("JewelStandaloneShowcase")
```

Comment:
Likely a copy-paste error

```suggestion
                    .getInstance(this::java.class)
```

Or also using the proposed `myLogger()`

Reply:
Not done yet